### PR TITLE
New beamer filetype

### PR DESCRIPTION
**Pull request by manolomartinez**

### CODE REVIEW

**Comment by manolomartinez** [comment]
Creates a new `beamer` filetype that simply clones `pandoc`. This is useful so as to have separate `pandoc#command#autoexec_command`, as per [this issue](https://github.com/vim-pandoc/vim-pandoc/issues/398).

File: ftplugin/beamer.vim [hunk 1]
```diff
@@ -1 +1,11 @@
+" vim: set fdm=marker:
+
+" Pandoc: {{{1
+" load vim-pandoc {{{2
 runtime ftplugin/pandoc.vim
+
+" init vim-pandoc-after, if present {{{2
+try
+    call pandoc#after#Init()
```
Comment (fmoralesc) [inline ftplugin/beamer.vim:9]: This is needed to load the compatibility plugins from vim-pandoc-after, yes.

**Comment by fmoralesc** [comment]
LGTM. A small note in the docs would make it perfect - but I don't think it's essential ;)

**Comment by manolomartinez** [comment]
Let me get to it. I've never edited vim-style docs, but it shouldn't be too hard :)

**Comment by manolomartinez** [comment]
I've changed the bit about `ftdetect` and added it to `plugin/pandoc.vim`, which is, I think, the right way to do it. I have also modified the docs accordingly.

**Comment by fmoralesc** [comment]
LGTM. Gracias por la contribución!

**Pull request merged**